test/unittests/test_system.py

```diff
@@ -0,0 +1,108 @@
+import unittest
+
+
+class TestSystem(unittest.TestCase):
+    def test_is_running_from_module(self):
+        from ovos_utils.system import is_running_from_module
+        self.assertFalse(is_running_from_module("mycroft"))
+        self.assertTrue(is_running_from_module("unittest"))
+
+    def test_ntp_sync(self):
+        # TODO
+        pass
+
+    def test_system_shutdown(self):
+        # TODO
+        pass
+
+    def test_system_reboot(self):
+        # TODO
+        pass
+
+    def test_ssh_enable(self):
+        # TODO
+        pass
+
+    def test_ssh_disable(self):
+        # TODO
+        pass
+
+    def test_restart_mycroft_service(self):
+        # TODO
+        pass
+
+    def test_restart_service(self):
+        # TODO
+        pass
+
+    def test_enable_service(self):
+        # TODO
+        pass
+
+    def test_disable_service(self):
+        # TODO
+        pass
+
+    def test_check_service_active(self):
+        # TODO
+        pass
+
+    def test_set_root_path(self):
+        from ovos_utils.system import set_root_path
+        set_root_path("test")
+        from ovos_utils.system import _USER_DEFINED_ROOT
```

Comment:
could be a module property now ?

Reply:
I wasn't sure how much this is used, but I can test that change

Reply:
This doesn't actually change behavior, only adds a wrapper method since the property is evaluated upon import, not reference. Since it's already marked as an internal variable, I'll leave it